### ISSUE TITLE
Not readable file. Diabdat.mpq

### ISSUE THREAD

**Issue by ebolba** [d07RiV/diabloweb#162]
**Description:**
Hello. Actually nothing special. I was playing normally at night and from this morning gamę crashes.

**App version:**
DiabloWeb 1.0.39 (Retail)

**Error message:**
    
    NotReadableError: The requested file could not be read, typically due to permission problems that have occurred after a reference to a file was acquired.

**User agent:**

    Mozilla/5.0 (Linux; Android 9; moto e(6) plus) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.185 Mobile Safari/537.36

**Save file:**
Dont know how? Im trying to play on smartphone VIA web browser. Becouse devilutionx version from googleplay is unplayable.

**Comment by AJenbo** [comment]
There is now a version for Android which should provide a much better experience: https://play.google.com/store/apps/details?id=org.diasurgical.devilutionx

**Comment by imneckro** [comment]
@AJenbo  May i ask if there is a way to host diabdat.mpq instead of spawn.mpq?

**Comment by AJenbo** [comment]
No. The projects continued existence has relied on it only hosting the shareware. I'm not sure what your goal is here, but I ask that you do not use this project to make the full game available to people that do not own the original game.
